doc/barclamp.md

```diff
@@ -1356,3 +1356,123 @@ place) you will have to add additional setup code to
 and/or
 [qa_crowbarsetup.sh](https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crobarsetup.sh)
 to take care of your barclamp's needs.
+
+## High Availability (HA) Integration
+
+[This commit](https://github.com/crowbar/crowbar-openstack/pull/512/files) has
```

Comment:
Careful, this was my first HA enabled barclamp and needed lots of fixing, so this pull request is far from perfect. Here are some of the follow-up commits:

* https://github.com/crowbar/crowbar-openstack/commit/9433bc51cf525398abca8028c83527757069aa26 (Wrong catalog entries for Barbican in the HA case)
* https://github.com/crowbar/crowbar-openstack/commit/cebcc2d21745b8c2c54e5a92fb0cc77ff3f8f69d (Fixes haproxy listening interface)
* https://github.com/crowbar/crowbar-openstack/commit/67a5e1feb5d107a2d2b783ba3bdb02e01ba1d34e (Typos)
*https://github.com/crowbar/crowbar-openstack/commit/ fc5e44f1190663969549e7c16cdee76f55f77fee (use systemd service type rather than LSB service type pacemaker resource)
* https://github.com/crowbar/crowbar-openstack/commit/0729ec31c3485853fa0ef69c7261a72b4c104cd7 (sync marks missing from the original commit)
* https://github.com/crowbar/crowbar-openstack/commit/a082419bd2461d9a1df4390f9fc6894ba8e9da14 (in this case HA broke the non-HA scenario)

It's better to base this on current master and make the link permanent by linking to the current HEAD commit explicitly. You'll have to bend the prose around that of course and make it clear that the Barbican barclamp is the object of study, but it is better than referencing a faulty commit.

Maybe something like this: "in this section, we will use the Barbican barclamp as an example. You will find the barclamp's source code as we will describe and quote it throughout this [here](https://github.com/crowbar/crowbar-openstack/tree/9805c7ddb81ffbf6f2b79061c6ceb332659cd614/chef/cookbooks/barbican).